### ISSUE TITLE
Λείπει η φωτο  AMP_1123.jpg

### ISSUE THREAD

**Issue by dimitrisdervas** [dimitrisdervas/lilavillas#3]
Δε τη βρίσκω στα αρχεία.

ΑΝ μπορείς ανέβασέ τη εδώ

**Comment by rovithakis** [comment]
Ναι, δεν την έχω εδώ. Αναμένω το Νώντα...

**Comment by rovithakis** [comment]
![amp_1124](https://cloud.githubusercontent.com/assets/15343074/10820482/17dcecf4-7e56-11e5-8e13-983a6a2c4d63.jpg)

**Comment by rovithakis** [comment]
δεν ξέρω αν είναι μικρή

**Comment by rovithakis** [comment]
AMP1124

**Comment by dimitrisdervas** [comment]
Οπότε χρησιμοποιούμε τη AMP1124 σωστά?

**Comment by dimitrisdervas** [comment]
έβαλα την AMP1124

**Comment by rovithakis** [comment]
Σωστά,αλλά γιατί βλέπουμε μόνο ένα κομμάτι της σαν να έχει γίνει ζουμ?
Και αυτό γίνεται με όλες τις σελίδες ενώ δεν ήταν έτσι...
